src/LoginPage/LoginFormContainer.jsx

```diff
@@ -4,15 +4,15 @@ import { useDispatch, useSelector } from 'react-redux';
 
 import LoginForm from './LoginForm';
 
-import { setIsLoginState } from '../slice';
+import { setIsLogin } from '../slice';
 
 export default function LoginFormContainer() {
   const dispatch = useDispatch();
 
   const isLogin = useSelector((selector) => selector.isLogin);
```

Comment:
```suggestion
const isLogin = useSelector(({ isLogin }) => isLogin);
```
이것도 취향차이긴 한데 이렇게 작성하면 조금 더 간결해지긴 합니당!ㅎㅎ

Reply:
아? selector 가 slice 의 inintialState 객체기 때문에 객체 구조분해할당이 가능하군용
이 부분 생각하지 못했네요!
디스트럭처링해서 selector 도 사용해야겠습니다! 🙏🏼